example-schema/types/MainType.yaml

```diff
@@ -104,13 +104,15 @@ properties:
     type: MainType
     description: Younger siblings description.
     label: Younger siblings label
+    otherNodeName: youngerSibling
```

Comment:
As mentioned above (in the relationships yaml) the name of the property doesnt highlight that its naming the target of a self-reference and is therefore not required for every relationship.

Reply:
It might also help if this property was specified directly after the `type` property to highlight that its effect is to rename that type (yes, I know the order is not important syntactically but it could improve readability)